FirebaseStorage/Tests/Unit/StorageAuthorizerTests.swift

```diff
@@ -0,0 +1,266 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+@testable import FirebaseStorage
+import GTMSessionFetcherCore
+import SharedTestUtilities
+import XCTest
+
+class StorageAuthorizerTests: StorageTestHelpers {
+  var appCheckTokenSuccess: FIRAppCheckTokenResultFake?
+  var appCheckTokenError: FIRAppCheckTokenResultFake?
+  var fetcher: GTMSessionFetcher?
+  var fetcherService: GTMSessionFetcherService?
+  var auth: FIRAuthInteropFake?
+  var appCheck: FIRAppCheckFake?
```

Comment:
nit: More of a stylistic thing, but imo test variables that are initialized in `setUp` should all be defined as implicitly unwrapped optionals (with a `!` instead of `?`).

Reply:
Done.